recipe/common.php

```diff
@@ -91,7 +91,7 @@
 
 set('bin/composer', function () {
     if (commandExist('composer')) {
-        $composer = run('which composer')->toString();
+        $composer = '{{bin/php}} ' . run('which composer')->toString();
```

Comment:
@lewactwo Kamil, could you please update your PR with @mbrodala comment? It's about little duplication of `{{bin/php}} ` part.